examples/controllers/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.3"
+services:
+  db:
+    build:
+      context: ./Docker
+      dockerfile: db.Dockerfile
+    volumes:
+      - ./var/pgdata:/var/lib/postgresql/data
```

Comment:
It's my personal opinion, so take for what it is, an opinion. I prefer named volumes explicitly defined.
```
version: "3.3"
services:
  db:
    build:
      context: ./Docker
      dockerfile: db.Dockerfile
    volumes:
      - db-data:/var/lib/postgresql/data
    ports:
      - "5432:5432"

volumes:
  db-data:
```